src/libtoast/src/toast_tod_filter.cpp

```diff
@@ -161,6 +134,11 @@ void toast::filter_polynomial(int64_t order, size_t n, uint8_t * flags,
                 }
             }
         }
+
+        if (info != 0) {
+            std::cerr << "ERROR: DGELLS info = " << info << std::endl;
+        }
```

Comment:
Do we want to remove this debugging statement?  Or use the toast logging to make an error message and raise an exception pointing to the line number?  Like this:
```
if (info != 0) {
    auto log = toast::Logger::get();
    std::ostringstream o;
    o << "DGELLS:  " << ngood << "/" << scanlen << " good samples, order " << norder;
    o << " failed with info " << info;
    log.error(o.str().c_str(), TOAST_HERE());
    throw std::runtime_error(o.str().c_str());
}
```

Reply:
logging and throwing is much better. I adopted that.